.github/workflows/element-web.yaml

```diff
@@ -12,26 +12,55 @@ on:
         branches: [develop, master]
     repository_dispatch:
         types: [upstream-sdk-notify]
+
+    # support triggering from other workflows
+    workflow_call:
+        inputs:
+            react-sdk-repository:
+                type: string
+                required: true
+                description: "The name of the github repository to check out and build."
+            matrix-js-sdk-sha:
+                type: string
+                required: false
+                description: "The Git SHA of matrix-js-sdk to build against. By default, will use a matching branch name if it exists, or develop."
+
+    # support manual dispatch
+    workflow_dispatch:
+        inputs:
+            matrix-js-sdk-sha:
+                type: string
+                required: false
+                description: "The Git SHA of matrix-js-sdk to build against. By default, will use a matching branch name if it exists, or develop."
```

Comment:
This will clobber CI results on the develop branch

Reply:
I don't think the sha that gets checked out has any bearing on where CI reports go back to?